lib/poclu/misc.c

```diff
@@ -53,21 +55,20 @@ poclu_create_any_context ()
 
 cl_int
 poclu_get_any_device (cl_context *context, cl_device_id *device,
-                      cl_command_queue *queue)
+                      cl_command_queue *queue, cl_platform_id *platform)
```

Comment:
It might be nice to allow passing NULL at least to the platform parameter as it's often not needed after initializing the context. Then it would create one internally to populate the other inputs. Lots of test changes in this patch just add an unused temp variable for this call.

Reply:
allright